pkg/controller/stackconfigpolicy/controller_test.go

```diff
@@ -400,11 +474,81 @@ func TestReconcileStackConfigPolicy_Reconcile(t *testing.T) {
 				assert.Equal(t, 1, policy.Status.Resources)
 				assert.Equal(t, 1, policy.Status.Ready)
 				assert.Equal(t, policyv1alpha1.ReadyPhase, policy.Status.Phase)
+				var esSecret corev1.Secret
+				// Verify the config secret created by the stack config policy controller
+				err = r.Client.Get(context.Background(), types.NamespacedName{
+					Namespace: "ns",
+					Name:      esv1.StackConfigElasticsearchConfigSecretName(esFixture.Name),
+				}, &esSecret)
+				assert.NoError(t, err)
+				elasticSearchConfigJSONData, err := json.Marshal(policy.Spec.Elasticsearch.Config)
```

Comment:
```suggestion
				elasticsearchConfigJSONData, err := json.Marshal(policy.Spec.Elasticsearch.Config)
```

or `es` but never with capitalised S.